django/contrib/gis/db/backends/oracle/adapter.py

```diff
@@ -1,4 +1,14 @@
-from cx_Oracle import CLOB
+import warnings
+from django.utils.deprecation import RemovedInDjango51Warning
+
+try:
+    from oracledb import CLOB
+except ImportError:
+    from cx_Oracle import CLOB
+    warnings.warn(
+        "cx_Oracle is deprecated. Please use oracledb instead.",
+        RemovedInDjango51Warning,
+    )
```

Comment:
Hi, do I need to add this warning in files except than `django/db/backends/oracle/base.py`, such as `django/contrib/gis/db/backends/oracle/adapter.py`, `django/contrib/gis/db/backends/oracle/introspection.py` and `django/db/backends/oracle/introspection.py`?